README.md

```diff
@@ -2,14 +2,31 @@
 
 Glam is a theme for The Balance website put together by the wonderful FFXIV community
 
+## Install Hugo
+Follow the instruction on [gohugo.io](https://gohugo.io/getting-started/installing/) to install hugo on your machine.
+
 ## Testing the theme locally
 
+First install all node dependencies:
+
+```
+npm install
+```
+
 Just run the following:
 
 ```
-hugo server -s exampleSite/ --themesDir=../.. --disableFastRender
+npm run serve:hugo
 ```
 
 ## Using the editor locally
 
-cd to the `exampleSite` directory and run `npx netlify-cms-proxy-server` alongside the `hugo` command; afterwards, navigate to `http://localhost:1313/admin`
+Change `IS_LOCAL_BUILD` inside `cms/index.js` to `true`.  This makes sure, netlify cms is available for your local build.
```

Comment:
I know I'm nitpicking the docs, but this is important – what about setting an environment variable for this? Then you could do something like `yarn build-cms` or `yarn build-cmd:dev` (or the appropriate `npm run` substitution).